cli/cmd/jobs.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/kosyfrances/rundeck-zabbix/lib/jobs"
+)
+
+var prefix string
+
+// jobsCmd represents the jobs command
+var jobsCmd = &cobra.Command{
+	Use:   "jobs",
+	Short: "generate Rundeck jobs",
+	Long: `generate Rundeck jobs document in Yaml format with hosts' triggers' information from Zabbix
+
+	If a prefix is given, jobs will be generated from triggers with the given prefix.
+	Otherwise, jobs will be generated from all the triggers in Zabbix.
+
+	If a file is given, the generated jobs are appended to the file.
+	If the given file path does not exist, it gets created.
+	Otherwise, a jobs.yml file is generated in the current path.`,
+	Run: runJobs,
+}
+
+func init() {
+	generateCmd.AddCommand(jobsCmd)
+	jobsCmd.Flags().StringVar(&filePath, "file", "jobs.yml", "Path to file where generated Jobs will be written")
```

Comment:
Cobra already displays it by default to the user so there is no need repeating it again.
```✗ go run main.go generate jobs --help
generate Rundeck jobs document in Yaml format with hosts' triggers' information from Zabbix

        If a prefix is given, jobs will be generated from triggers with the given prefix.
        Otherwise, jobs will be generated from all the triggers in Zabbix.

        If a file is given, the generated jobs are appended to the file.
        If the given file path does not exist, it gets created.
        Otherwise, a jobs.yml file is generated in the current path.

Usage:
  cli generate jobs [flags]

Flags:
      --file string     Path to file where generated Jobs will be written. (default "jobs.yml")
  -h, --help            help for jobs
      --prefix string   Generate triggers from the given prefix, otherwise, jobs will be generated from all the triggers in Zabbix.

Global Flags:
      --config string   config file (default is $HOME/.cli.yaml)```